src/gui/FxMixerView.cpp

```diff
@@ -185,6 +185,8 @@ void FxMixerView::addNewChannel()
 	updateFxLine(newChannelIndex);
```

Comment:
No, you're justification is completely valid in my opinion.  I didn't realize the nomenclature was already there depsite having the audacity to comment under a line you hadn't change. :hamburger: